project/plugins.sbt

```diff
@@ -1,7 +1,16 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
 
-import CakePluginDependencies._
+import net.cakesolutions.CakePlatformDependencies.SbtDependencies._
```

Comment:
To properly reduce this copying, we would need to define and publish an additional SBT plugin. In this case, I think this would be potential overkill?